TYPO3.Flow/Classes/TYPO3/Flow/Package/PackageManager.php

```diff
@@ -375,61 +351,15 @@ protected function filterPackagesByPath(&$packages, $filterPath)
      */
     protected function filterPackagesByType(&$packages, $packageType)
     {
-        $filteredPackages = array();
+        $filteredPackages = [];
         /** @var $package Package */
         foreach ($packages as $package) {
             if ($package->getComposerManifest('type') === $packageType) {
                 $filteredPackages[$package->getPackageKey()] = $package;
             }
         }
-        return $filteredPackages;
-    }
-
-    /**
-     * Returns the upper camel cased version of the given package key or FALSE
-     * if no such package is available.
-     *
-     * @param string $unknownCasedPackageKey The package key to convert
-     * @return mixed The upper camel cased package key or FALSE if no such package exists
-     * @api
-     */
-    public function getCaseSensitivePackageKey($unknownCasedPackageKey)
-    {
-        $lowerCasedPackageKey = strtolower($unknownCasedPackageKey);
-        return (isset($this->packageKeys[$lowerCasedPackageKey])) ? $this->packageKeys[$lowerCasedPackageKey] : false;
-    }
 
-    /**
-     * Resolves a Flow package key from a composer package name.
-     *
-     * @param string $composerName
-     * @return string
-     * @throws Exception\InvalidPackageStateException
-     */
-    public function getPackageKeyFromComposerName($composerName)
-    {
-        if (count($this->composerNameToPackageKeyMap) === 0) {
-            foreach ($this->packageStatesConfiguration['packages'] as $packageKey => $packageStateConfiguration) {
-                $this->composerNameToPackageKeyMap[strtolower($packageStateConfiguration['composerName'])] = $packageKey;
-            }
-        }
-        $lowercasedComposerName = strtolower($composerName);
-        if (!isset($this->composerNameToPackageKeyMap[$lowercasedComposerName])) {
-            throw new \TYPO3\Flow\Package\Exception\InvalidPackageStateException('Could not find package with composer name "' . $composerName . '" in PackageStates configuration.', 1352320649);
-        }
-        return $this->composerNameToPackageKeyMap[$lowercasedComposerName];
-    }
-
-    /**
-     * Check the conformance of the given package key
-     *
-     * @param string $packageKey The package key to validate
-     * @return boolean If the package key is valid, returns TRUE otherwise FALSE
-     * @api
```

Comment:
What about this function, it's @api but is just removed. Would need a @deprecated b/c dummy at least